rs/registry/keys/src/lib.rs

```diff
@@ -34,7 +34,7 @@ pub const CRYPTO_THRESHOLD_SIGNING_KEY_PREFIX: &str = "crypto_threshold_signing_
 pub const DATA_CENTER_KEY_PREFIX: &str = "data_center_record_";
 pub const ECDSA_SIGNING_SUBNET_LIST_KEY_PREFIX: &str = "key_id_";
 pub const CHAIN_KEY_ENABLED_SUBNET_LIST_KEY_PREFIX: &str = "master_public_key_id_";
-pub const CANISTER_RANGES_PREFIX: &str = "canister_ranges_";
+pub const CANISTER_RANGE_PREFIX: &str = "canister_range_";
```

Comment:
Wouldn't it result in too many registry values if we store one canister range per registry value?

Reply:
A related discussion: https://docs.google.com/document/d/1H_BoIJU8QIRj5J8UJFeYVsaW_OZrBfYtKhkr0jvQgEs/edit?disco=AAABjNMq8Dg